controller/app.controller.js

```diff
@@ -24,10 +24,11 @@ exports.getAllApi = (req,res,next) => {
 
 
 exports.getArticleById = (req,res,next) => {
-  
-     const  {id} = req.params
-  selectArticleById(id).then((articleObj) => {
+ 
+     const {article_id} = req.params
+     
+      selectArticleById(article_id).then((articleObj) => {
     res.status(200).send(articleObj)
```

Comment:
Instead of sending the article back directly on the body here it would be nice to give this a descriptive name (this will help future you in frontend 😄 )